### ISSUE TITLE
Disable Packages that fail to install

### ISSUE THREAD

**Issue by rodonn** [wsdjeg/SpaceVim#1069]
Everytime I start Spacevim I get errors about packages failing to install. I would like to disable those packages so that Spacevim is no longer slowed down trying to unsucessfully install them.

Is there an easy way to do so? I would be nice to document a solution.

## Environment Information
- OS: OSX 10.12.6
- vim version: VIM - Vi IMproved 8.0 (2016 Sep 12, compiled Nov 28 2017 06:16:33)

## The reproduce ways from Vim starting (Required!)
Happens every time I start vim. I can use vim after the errors load, but it takes a few seconds for the installs to fail, so it is annoying..

## Screenshots

![image](https://user-images.githubusercontent.com/2594443/33542823-71f32430-d889-11e7-9303-1efba655dcb0.png)

**Comment by wsdjeg** [comment]
> Everytime I start Spacevim I get errors

what is the reproduce step from command line?

**Comment by wsdjeg** [comment]
before I fix this issue, I want to know why it is failed? I will try to post a debug step tonight.

**Comment by wsdjeg** [comment]
@rodonn what about run `:SPUpdate ag.vim`? I just test it ,but it works well for me in vim and neovim.

**Comment by Shougo** [comment]
It is the bug of dein.vim.
Please update dein.vim.

**Comment by wsdjeg** [comment]
same as #1070

**Comment by rodonn** [comment]
Thank you for your help!

I get these errors everytime I open vim from the command line. (i.e. I call `vim`)

I deleted my `.Spacemacs` folder and my `.cache/vimfiles` and reinstalled using `curl -sLf https://spacevim.org/install.sh | bash`.

After opening vim by calling `vim` from the command line, vim reinstalled all the spacevim packages, except for the same list as I was getting before.
i.e. I cannot seem to install TaskList, ack, ag, php, pyte, taglist, vim-coffee-script, twilight256, or wombat256.

I ran `:SPUpdate dein` and it reports that it was updated. 
If I run `:SPUpdate ag` I see a window that says "Updating plugins (0/1)", but then it doesn't make any progress. 
The :messages buffer looks like

> Messages maintainer: Bram Moolenaar <Bram@vim.org>
> Error detected while processing function 85[2]..<SNR>178_on_install_stdout:
> line   10:
> E716: Key not present in Dictionary: 4
> E116: Invalid arguments for function <SNR>178_msg_on_install_process
> Error detected while processing function 82[2]..<SNR>178_on_install_stdout:
> line   10:
> E716: Key not present in Dictionary: 3
> E116: Invalid arguments for function <SNR>178_msg_on_install_process


I'm ok with not having the packages that are failing to install, I just want to stop Spacevim from repeatedly trying to install them every time it opens (since it means that vim now takes more than a second to open)
